### PR TITLE
Provided default values for migrate.py #2

### PR DESCRIPTION
**Pull request by enkidulan**
Solution
---------

**Original issue and supporting materials**: #2 Provide default values for `migrate.py` parameters.
**Downsides**: None

Roll-out plan
-------------

**Deployment risks**: deployment risks are minimal, but delivery requires releasing new versions of **EBN** and **DMON**
**Deployment assistance**: It's going to be my first deployment, so I need some one by my side in case of emergency. 
**Backing services related changes**: the only required change is to upgrade `pyramid_heroku` package version on **EBN** and **DMON** and redeploy those services.
**Backuping steps**: Heroku will take care of it automatically 

**Deployment steps**:
  1. [ ] Make new release of `pyramid_heroku` 
  2. [ ] Create correlating PR to **DMON**.
  3. [ ] Create correlating PR to **EBN**.
  4. [ ] Make new release of **EBN** and deploy it.
  5. [ ] Verify **EBN** deployment
  6. [ ] Make new release of **DMON** and deploy it.
  7. [ ] Verify **DMON** deployment

Verification plan
------------------
   
  1. [ ] Monitor in **EBN** and **DMON** papertrail logs that there are no error messages during deployment,
  2. [ ] Make sure that during deployment of **EBN** `migration.py` module was called, 
  3. [ ] Successful deploy procedure. Changes are related to releasing workflow, so new version won't be released in case of error, see [Procfile](https://github.com/niteoweb/ebn/blob/master/Procfile#L4>)

Rollback strategy
-----------------
 
  1. [ ] Rollback **EBN** Heroku app. 
  2. [ ] Rollback **DMON** Heroku app.

### CODE REVIEW

File: pyramid_heroku/migrate.py [hunk 1]
```diff
@@ -157,19 +157,20 @@ def migrate(self):
 
 def main():
     parser = argparse.ArgumentParser(
-        usage='python -m pyramid_heroku.migrate my_app etc/production.ini app:main',
+        usage='python -m pyramid_heroku.migrate my_app --ini_file=etc/production.ini --app_section=app:main',
```
Comment (zupo) [inline pyramid_heroku/migrate.py]: Is this change needed?

Comment (enkidulan) [inline pyramid_heroku/migrate.py]: @zupo keyword arguments with default values is a native way to implement optional parameters. Consider a case when a user wants to specify only one optional value:

    # using defaults for both `ini_file` and `app_section`
    $ python -m pyramid_heroku.migrate my_app 

    # using default value for `app_section`
    $ python -m pyramid_heroku.migrate my_app --ini_file=etc/production.ini

    # using default value for `ini_file`
    $ python -m pyramid_heroku.migrate my_app --app_section=app:main

It's possible to achieve similar behavior with using positional arguments, but it will make CLI more confusing:
    
    # using defaults for both `ini_file` and `app_section`
    $ python -m pyramid_heroku.migrate my_app 

    # using default value for `app_section`
    $ python -m pyramid_heroku.migrate my_app etc/production.ini

    # using defaul for `ini_file` when user wants to provide value for `app_section` is not possible because of positional nature of arguments
    $ python -m pyramid_heroku.migrate my_app  etc/production.ini  app:main

**Comment by zupo** [comment]
What is your PyPI handle? So I can give you permissions to release a new version.

**Comment by enkidulan** [comment]
my username on PyPI is enkidu

**Comment by zupo** [comment]
granted
